### PR TITLE
Team merge

### PR DESCRIPTION
**Pull request by SueSmith**
Not sure why the changes pushed to a new branch, something to do with collaborator status on the fork/branch maybe..

Anyway @carolin-postman and @claireriley could you give this a once over? cc @lashcrafty

### CODE REVIEW

File: src/pages/docs/administration/team-merge.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+You can bulk export your admin personal workspaces or your personal workspaces plus the workspaces of any team you have joined.
+
+You ___should___ choose centralized migration if the following conditions apply:
+
+* Your organization prefers to have one person perform the migration and collection organization into the new team.
+
+You should ___not___ choose distributed migration if your organization relies on the following:
+
+* Postman-published documentation. URLs would need to be republished from your new team instance.
+* Mocks and monitors—you would need to be recreate these.
```
Comment (carolin-postman) [inline src/pages/docs/administration/team-merge.md:71]: remove the extra "be"

**Review by carolin-postman — CHANGES_REQUESTED** [review]
Thank you for reviewing @SueSmith. Just found something minor.

File: src/pages/docs/administration/team-merge.md (hunk 1 above)
Comment (SueSmith) [reply src/pages/docs/administration/team-merge.md]: gah sorted

**Comment by carolin-postman** [comment]
@claireriley - can this be reviewed and published by latest EOW? Let me know what I can do to help move this along. Thanks. 
cc @SueSmith

File: src/pages/docs/administration/team-merge.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+## Migration FAQs
+
+* **I'm on a free account, what happens to my personal collections once I join a new team?**
+As long as you are synced to the Postman cloud, the organizational structure of your personal workspace collections and environments will be carried over into the new team. Anything that has been shared to a team workspace will remain with the old team after you leave, meaning you will lose access to it. If you experience any issues when joining a new team [contact Postman support](postman.com/support).
+
+* **Can I be on two teams at once?**
+No, each account can be on one Postman team at a time. Multi-team collaboration and guest accounts are on [Postman's roadmap](https://trello.com/b/4N7PnHAz/postman-roadmap-for-developers).
+
+* **What happens to published documentation from my previous team?**
+Your previous links will break since the old team is disabled. New documentation URLs will need to be generated when collections are [re-published from your new team pages](/docs/publishing-your-api/publishing-your-docs/).
```
Comment (claireriley) [inline src/pages/docs/administration/team-merge.md:131]: @carolin-postman do admins need to remove custom domains from their old teams prior to moving to a new team if they want to use the same URL for same docs? will there be an issue if they don't?

File: src/pages/docs/administration/team-merge.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+* **What happens to the old team once everyone leaves?**
+Once the last person has left, the team is automatically disabled. If there is only one admin remaining on the team, they will automatically inherit all the team workspaces data (collections and environments) in which they can perform the bulk export.
```
Comment (claireriley) [inline src/pages/docs/administration/team-merge.md:134]: "in which they" -> "and" ?

File: src/pages/docs/administration/team-merge.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+* **Will my monitors / mocks transfer over?**
+No, you will need to [recreate any monitors](/docs/designing-and-developing-your-api/monitoring-your-api/setting-up-monitor/) and [mocks](docs/designing-and-developing-your-api/mocking-data/setting-up-mock/) in your new team.
+
+* **What data is exported in a large JSON file data-dump?**
+All collections in your personal workspace and any workspaces that you have joined, in addition to environments, globals, and header presets, are included when you export a data-dump.
```
Comment (claireriley) [inline src/pages/docs/administration/team-merge.md:140]: We could link /docs/getting-started/importing-and-exporting-data/#exporting-data-dumps again here

File: src/pages/docs/administration/team-merge.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+
+* **What data does NOT get exported?** You will need to [reassign roles (workspace, collection, and environment level)](/docs/collaborating-in-postman/roles-and-permissions/), [recreate workspaces](/docs/collaborating-in-postman/collaboration-intro/#creating-a-new-workspace-from-the-menu), [recreate monitors](/docs/designing-and-developing-your-api/monitoring-your-api/setting-up-monitor/), [recreate mocks](/docs/designing-and-developing-your-api/mocking-data/setting-up-mock/), [re-import API schemas](/docs/getting-started/importing-and-exporting-data/#importing-api-specifications), and [re-publish documentation from your new team pages](/docs/publishing-your-api/publishing-your-docs/).
+
+> Feel free to reach out to [Postman support](http://postman.com/support) if you have any questions or run into any issues setting up Postman for your team.
```
Comment (claireriley) [inline src/pages/docs/administration/team-merge.md:144]: I'd remove "Feel free to"

File: src/pages/docs/administration/team-merge.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+
+## Next Steps
+
+[Configure SSO](/docs/administration/sso/admin-sso/) if you are subscribed to the Business or Enterprise plan.
```
Comment (claireriley) [inline src/pages/docs/administration/team-merge.md:148]: "to the Business or Enterprise plan" -> "to a Postman Business or Enterprise plan"

**Review by claireriley — CHANGES_REQUESTED** [review]
@SueSmith let me know if I can help with edits!

File: src/pages/docs/administration/team-merge.md (hunk 2 above)
Comment (carolin-postman) [reply src/pages/docs/administration/team-merge.md:131]: @claireriley  - Good question. 

If they are moving to a new team and want to take the domain with them then they have to remove the domain and change it to anything else for it to be free for them to claim on the new team. 

Feel free to reword that and include in the next version.

**Comment by SueSmith** [comment]
@carolin-postman sorry this keeps slipping down the list.. I've added you as a collaborator to the repo, can you access this pr now?

**Comment by carolin-postman** [comment]
@SueSmith I can access now, that did it, thank you! I managed to make some of the smaller edits. I trust you and @claireriley on modifying the rest as you see fit and publishing afterwards. Thanks again for your help.

**Comment by SueSmith** [comment]
@claireriley almost all resolved with a handful of little outstanding things, think you could take a last pass so we can get this published?

**Comment by carolin-postman** [comment]
@claireriley @SueSmith - just checking in on this.

**Review by claireriley — APPROVED** [review]

**Comment by claireriley** [comment]
@SueSmith @carolin-postman made a couple last changes and merged—it'll be published early this week with the v2 and public workspaces releases!

**Comment by carolin-postman** [comment]
Thanks @claireriley !

**Pull request merged**